plugins/fluentd_telemetry_plugin/conf/fluentd_telemetry_plugin.cfg

```diff
@@ -6,6 +6,7 @@ interval = 30
 message_tag_name =
 xdr_mode=False
 xdr_ports_types=legacy;aggregated;plane
+timeout=60
```

Comment:
did you try to set a timeout for multiple endpoints?
did you try to get/set the timeout via the REST API? you should add the new parameter to `set_conf.schema.json`
all telemetry parameters are , separated for multiple endpoints

Reply:
following to our conversation, please add `telemetry_request_timeout` field under the `streaming` section, so we will have one timeout for all endpoints, instead of adding the timeout under the `ufm-telemetry-endpoint`

Reply:
done